huaweicloud/services/acceptance/servicestage/resource_huaweicloud_servicestage_environment_test.go

```diff
@@ -0,0 +1,582 @@
+
+  persistence {
+    type = "HTTP_COOKIE"
+  }
+}
+
+resource "huaweicloud_elb_monitor" "test" {
+  count = 2
+
+  protocol    = "HTTP"
+  interval    = 20
+  timeout     = 15
+  max_retries = 10
+  url_path    = "/"
+  port        = 8080
+  pool_id     = huaweicloud_elb_pool.test[count.index].id
+}
+
+resource "huaweicloud_elb_member" "test" {
+  count = 2
+
+  address       = huaweicloud_compute_instance.test[count.index].access_ip_v4
+  protocol_port = 8080
+  pool_id       = huaweicloud_elb_pool.test[count.index].id
+  subnet_id     = huaweicloud_vpc_subnet.test.subnet_id
+}
+
+resource "huaweicloud_vpc_eip" "test" {
+  count = 2
+
+  publicip {
+    type = "5_bgp"
+  }
+
+  bandwidth {
+    share_type  = "PER"
+    name        = "%[1]s_${count.index}"
+    size        = 10
+    charge_mode = "traffic"
+  }
+}
+
+resource "huaweicloud_rds_instance" "test" {
+  count = 2
+
+  name              = "%[1]s_${count.index}"
+  flavor            = "rds.pg.n1.large.2"
+  availability_zone = [data.huaweicloud_availability_zones.test.names[0]]
+  security_group_id = huaweicloud_networking_secgroup.test.id
+  subnet_id         =  huaweicloud_vpc_subnet.test.id
+  vpc_id            = huaweicloud_vpc.test.id
+  time_zone         = "UTC+08:00"
+  fixed_ip          = var.rds_config[count.index].fixed_ip
+
+  db {
+    password = "Huawei##123"
+    type     = "PostgreSQL"
+    version  = "12"
+    port     = var.rds_config[count.index].port
+  }
+
+  volume {
+    type = "CLOUDSSD"
+    size = 50
+  }
+
+  backup_strategy {
+    start_time = "08:00-09:00"
+    keep_days  = 1
+  }
+}
+
+resource "huaweicloud_dcs_instance" "test" {
+  count = 2
+
+  name               = "%[1]s_${count.index}"
+  engine_version     = "5.0"
+  password           = "Huawei##123"
+  engine             = "Redis"
+  port               = var.dcs_config[count.index].port
+  capacity           = 0.125
+  vpc_id             = huaweicloud_vpc.test.id
+  subnet_id          = huaweicloud_vpc_subnet.test.id
+  availability_zones = [data.huaweicloud_availability_zones.test.names[0]]
+  flavor             = "redis.ha.xu1.tiny.r2.128"
+  maintain_begin     = "22:00:00"
+  maintain_end       = "02:00:00"
+
+  backup_policy {
+    backup_type = "auto"
+    begin_at    = "00:00-01:00"
+    period_type = "weekly"
+    backup_at   = [4]
+    save_days   = 1
+  }
+}
+`, rName)
+}
+
+func testAccEnvironment_basic(rName string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_servicestage_environment" "test" {
+  name        = "%s"
+  description = "Created by terraform test"
+  vpc_id      = huaweicloud_vpc.test.id
+
+  basic_resources {
+    type = "cce"
+    id   = huaweicloud_cce_cluster.test[0].id
+  }
+  basic_resources {
+    type = "cci"
+    id   = huaweicloud_cci_namespace.test[0].name
+  }
+  basic_resources {
+    type = "ecs"
+    id   = huaweicloud_compute_instance.test[0].id
+  }
+  basic_resources {
+    type = "cce"
+    id   = huaweicloud_as_group.test[0].id
+  }
```

Comment:
the type and id do not match, please check it

Reply:
fixed.